config/sample-macros.cfg

```diff
@@ -10,7 +10,9 @@
 ######################################################################
 
 # Replace the slicer's custom start and end g-code scripts with
-# START_PRINT and END_PRINT.
+# START_PRINT and END_PRINT. Examples for using the variable to
+# parameter pass through in various slicers is documented in
+# docs/Slicers.md
 
```

Comment:
FWIW, I'm not sure I understand the new comment.  How about: `See docs/Slicers.md for more information on using these macros.`